### PR TITLE
feat: migrate less to token for Image

### PR DESCRIPTION
**Pull request by guan404ming**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

ref #41884 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      migrate less to token for Image     |
| 🇨🇳 Chinese |     Image 组件less变量迁移      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 4091d56</samp>

Refactor the style tokens for the `Image` and `PreviewGroup` components and update the documentation files to explain how to migrate the less variables to the component tokens.

### <samp>🤖 Generated by Copilot at 4091d56</samp>

*  Add new component tokens for image preview operation buttons ([link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-922f1030835ae207735e4fda28ea766dc4a4e35ae7ba47db9d084a4c0b2fec3eR11-R13), [link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-999566baa64628c49813370e7cb13c3c8f5e0c99ef8fe9cec7c5052985c10d9aR39-R51), [link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-dff3e0a0a5fae848621e55da4e2adaf0ec6c57d5e29d3f601799c8898c280739R39-R51))
   *  Extend `ComponentToken` interface with `imagePreviewOperationSize`, `imagePreviewOperationColor`, and `imagePreviewOperationDisabledColor` properties in `components/image/style/index.ts` ([link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-922f1030835ae207735e4fda28ea766dc4a4e35ae7ba47db9d084a4c0b2fec3eR11-R13))
   *  Document the migration of less variables to component tokens for `Image` component in `docs/react/migrate-less-variables.en-US.md` and `docs/react/migrate-less-variables.zh-CN.md` ([link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-999566baa64628c49813370e7cb13c3c8f5e0c99ef8fe9cec7c5052985c10d9aR39-R51), [link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-dff3e0a0a5fae848621e55da4e2adaf0ec6c57d5e29d3f601799c8898c280739R39-R51))
*  Remove redundant properties from `ImageToken` interface ([link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-922f1030835ae207735e4fda28ea766dc4a4e35ae7ba47db9d084a4c0b2fec3eL16-R19))
   *  Delete `imagePreviewOperationSize`, `imagePreviewOperationColor`, and `imagePreviewOperationDisabledColor` properties from `ImageToken` interface in `components/image/style/index.ts` ([link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-922f1030835ae207735e4fda28ea766dc4a4e35ae7ba47db9d084a4c0b2fec3eL16-R19))
*  Simplify and update the creation of `imageToken` object ([link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-922f1030835ae207735e4fda28ea766dc4a4e35ae7ba47db9d084a4c0b2fec3eL301-R305), [link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-922f1030835ae207735e4fda28ea766dc4a4e35ae7ba47db9d084a4c0b2fec3eR318-R322))
   *  Remove the logic that calculates `imagePreviewOperationColor` and `imagePreviewOperationDisabledColor` from `imagePreviewOperationColor` property of `token` object in `components/image/style/index.ts` ([link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-922f1030835ae207735e4fda28ea766dc4a4e35ae7ba47db9d084a4c0b2fec3eL301-R305))
   *  Assign `imagePreviewOperationColor`, `imagePreviewOperationDisabledColor`, and `imagePreviewOperationSize` properties to `token` object using `mergeToken` function in `components/image/style/index.ts` ([link](https://github.com/ant-design/ant-design/pull/42048/files?diff=unified&w=0#diff-922f1030835ae207735e4fda28ea766dc4a4e35ae7ba47db9d084a4c0b2fec3eR318-R322))

### CODE REVIEW

File: docs/react/migrate-less-variables.zh-CN.md [hunk 1]
```diff
@@ -36,3 +36,16 @@ title: less 变量迁移 Component Token
 | `@modal-confirm-body-padding` | - | 由于样式变化已废弃 |
 | `@modal-confirm-title-font-size` | `modalHeaderTitleFontSize` | - |
 | `@modal-border-radius` | `borderRadiusLG` | - |
+
+## Image 图片
+
+<!-- prettier-ignore -->
+| less 变量 | Component Token | 备注 |
+| --- | --- | --- |
+| `@image-size-base` | - | 由于样式变化已废弃 |
```
[the remark this answers is not in the document]
Comment (guan404ming) [reply docs/react/migrate-less-variables.zh-CN.md]: 我已經修正了以上錯誤，再麻煩大佬看看～

**Comment by yoyo837** [comment]
Hey guy, Please rebase.

**Comment by guan404ming** [comment]
> Hey guy, Please rebase.
Sorry for not really familiar with rebase. Could you please provide some suggestion or help me to make it correct? Thank you very much!!

**Comment by yoyo837** [comment]
```shell
git remote add upstream git@github.com:ant-design/ant-design.git

git fetch --all

git rebase upstream/feature
# And resolve conflicts manually

git push -f
```

**Comment by yoyo837** [comment]
Try `git push -f` or `git push --force`

**Comment by guan404ming** [comment]
Oh no! I messed up. So sorry for it and I would try my best to fix it.

**Comment by guan404ming** [comment]
Not sure this is correct or not @yoyo837 ~

**Comment by yoyo837** [comment]
> Not sure this is correct or not @yoyo837 ~

Everything looks fine.

**Comment by guan404ming** [comment]
> > Not sure this is correct or not @yoyo837 ~
> 
> Everything looks fine.

Thanks very much to spent lots of time helping me deal with this problem!! 🙇

File: docs/react/migrate-less-variables.zh-CN.md [hunk 2]
```diff
@@ -149,7 +149,18 @@ title: Less 变量迁移 Design Token
...
+<!-- prettier-ignore -->
+| less 变量 | Component Token | 备注 |
+| --- | --- | --- |
+| `@image-size-base` | - | 未使用已废弃 |
+| `@image-font-size-base` | - | 未使用已废弃 |
+| `@image-bg` | `colorFillTertiary` | - |
+| `@image-color` | `colorTextLightSolid` | - |
+| `@image-preview-operation-size` | `imagePreviewOperationSize` | - |
+| `@image-preview-operation-color` | `imagePreviewOperationColor` | - |
+| `@image-preview-operation-disabled-color` | `imagePreviewOperationDisabledColor` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:163]:
```suggestion
| `@image-preview-operation-size` | `previewOperationSize` | - |
| `@image-preview-operation-color` | `previewOperationColor` | - |
| `@image-preview-operation-disabled-color` | `previewOperationColorDisabled` | - |
```
命名调整一下哈

File: components/image/demo/component-token.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { ConfigProvider, Image } from 'antd';
+import React from 'react';
+
+const App: React.FC = () => (
+  <ConfigProvider
+    theme={{
+      components: {
+        Image: {
+          colorFillTertiary: '#b20000',
+          colorTextLightSolid: '#222',
```
Comment (MadCcc) [inline components/image/demo/component-token.tsx:10]: 这两个不是组件 token，可以不用在 demo 里展示

Comment (guan404ming) [reply components/image/demo/component-token.tsx]: 完成了，再請大佬看看～

File: components/image/style/index.ts [hunk 4]
```diff
@@ -323,5 +317,10 @@ export default genComponentStyleHook(
   },
   (token) => ({
     zIndexPopup: token.zIndexPopupBase + 80,
+    imagePreviewOperationColor: new TinyColor(token.colorTextLightSolid).toRgbString(),
+    imagePreviewOperationDisabledColor: new TinyColor(token.colorTextLightSolid)
+      .setAlpha(0.25)
+      .toRgbString(),
+    imagePreviewOperationSize: token.fontSizeIcon * 1.5, // FIXME: fontSizeIconLG
```
Comment (MadCcc) [inline components/image/style/index.ts:324]: 代码里也更新一下哈

Comment (guan404ming) [reply components/image/style/index.ts]: 剛剛漏掉了，再麻煩大佬看看～

Comment (guan404ming) [reply components/image/style/index.ts]: @MadCcc 再麻煩大佬 CR 了～

**Review by MadCcc — APPROVED** [review]

**Pull request merged**